ACCStatsUploader/Converters/BrakeBalanceConverter.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ACCStatsUploader.Converters {
+    public class BrakeBalanceConverter {
+        private static Dictionary<CarModel.KUNOS_ID, int> brakeBalanceOffset = new Dictionary<CarModel.KUNOS_ID, int> {
```

Comment:
replace this dictionary with an extension method for the enumeration KUNOS_ID.
https://stackoverflow.com/a/5985710